flutter_app/lib/screens/home_screen.dart

```diff
@@ -1,27 +1,35 @@
 import 'package:flutter/material.dart';
+import 'package:imacs/modules/get_airside_logs.dart';
 import 'package:imacs/modules/mavlink_communication.dart';
 import 'package:imacs/modules/get_drone_information.dart';
 import 'package:imacs/widgets/drone_information_widget.dart';
+import 'package:imacs/widgets/display_all_logs_widget.dart';
 
 class HomePage extends StatelessWidget {
   HomePage({Key? key, required this.title}) : super(key: key);
 
   final String title;
   final comm =
       MavlinkCommunication(MavlinkCommunicationType.tcp, '127.0.0.1', 14550);
+  final String pathToDirectory = "C:\\Users\\emmao\\Documents\\logs";
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         title: Text(title),
       ),
-      body: Column(
-        children: [
-          DroneInformation(
-            getDroneInformation: GetDroneInformation(comm: comm),
-          ),
-        ],
+      body: SingleChildScrollView(
```

Comment:
I would caution agains using this widget when it is nested with itself since it can lead to stuff like this
![image](https://github.com/UWARG/IMACS-2/assets/67008243/cfd5067a-b5ff-4afc-985b-93505bdf054f)

Don't worry how home_screen looks right now, we still haven't finalized the layout or anything, we're just getting the widgets to a usable state atm